connection.go

```diff
@@ -124,12 +132,26 @@ func (c *Connection) Serve() {
 		c.Close()
 	}()
 
+	remoteAddr := trimStringFromSym(c.Conn.RemoteAddr().String())
+	strs := strings.Split(remoteAddr, ".")
```

Comment:
is this address always dns-resolved? If that's an IP, there could be a weird value instead.